pkg/nsx/services/nsxserviceaccount/cluster.go

```diff
@@ -482,6 +469,31 @@ func (s *NSXServiceAccountService) updatePIAndCCPCert(normalizedClusterName, uid
 	return nil
 }
 
+func (s *NSXServiceAccountService) updatePIAndCCPCert(normalizedClusterName, uid, cert string) error {
```

Comment:
Move this func prior to `updatePICert` to avoid many lines changed.

Reply:
done